runelite-client/src/main/java/net/runelite/client/plugins/screenshot/ScreenshotPlugin.java

```diff
@@ -306,6 +306,10 @@ else if (player != client.getLocalPlayer() && (player.isFriendsChatMember() || p
 			{
 				takeScreenshot("Death " + player.getName(), SD_DEATHS);
 			}
+			else if (player != client.getLocalPlayer() && player.isClanMember() && config.screenshotClanDeath() && player.getCanvasTilePoly() != null)
```

Comment:
This can be merged into the above statement.

Reply:
Should be good now, nice spot

Reply:
@raiyni ok finally got around to fixing this